packages/core/src/payment-integration/default-payment-integration-service.ts

```diff
@@ -207,4 +209,16 @@ export default class DefaultPaymentIntegrationService implements PaymentIntegrat
 
         return this._storeProjection.getState();
     }
+
+    async updateProviderCustomerData<T>(
+        providerCustomerData: T,
+    ): Promise<PaymentIntegrationSelectors> {
+        await this._store.dispatch(
```

Comment:
Is this supposed to be async?

Reply:
It does not matter which action get called, however `this._store.dispatch` returns a Promise. So yes, we should use `await` here.